javav2/example_code/appsync/src/main/java/com/example/appsync/ListApiKeys.java

```diff
@@ -0,0 +1,69 @@
+// snippet-sourcedescription:[ListApiKeys.java demonstrates how to get API keys.]
+//snippet-keyword:[AWS SDK for Java v2]
+// snippet-service:[AWS AppSync]
+// snippet-keyword:[Code Sample]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[04-01-2022]
+// snippet-sourceauthor:[scmacdon - AWS]
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.appsync;
+
+//snippet-start:[appsync.java.get_keys.import]
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.appsync.AppSyncClient;
+import software.amazon.awssdk.services.appsync.model.ApiKey;
+import software.amazon.awssdk.services.appsync.model.AppSyncException;
+import software.amazon.awssdk.services.appsync.model.ListApiKeysRequest;
+import software.amazon.awssdk.services.appsync.model.ListApiKeysResponse;
+import java.util.List;
+//snippet-end:[appsync.java.get_keys.import]
+
+public class ListApiKeys {
+
+    public static void main(String[] args) {
+
+        final String usage = "\n" +
+                "Usage: " +
+                "   <apiId> \n\n" +
+                "Where:\n" +
+                "   apiId - The id of the API (You can obtain the value from the AWS Management console). \n\n" ;

```

Comment:
(You can obtain the value from the AWS Management console). --> (You can get this value from the AWS Management Console).

Reply:
fixed